### DISCUSSION TITLE
Dropping ENABLE_CHARTS

### DISCUSSION THREAD

**Discussion by maliberty** [The-OpenROAD-Project/OpenROAD#6301] [General]
Now that centos7 is dropped I would like to drop the ENABLE_CHARTS option.  I believe qtcharts is available on all supported platforms.  Does anyone have an objection?  @gadfort @donn @QuantamHD

**Comment by QuantamHD** [comment]
Lgtm

**Comment by gadfort** [comment]
No issues for me

**Comment by donn** [comment]
no objections
